ci/jjb_service_job.yml

```diff
@@ -0,0 +1,22 @@
+- job:
+    name: 'centos-container-pipeline-service-job-openshift'
+    description: |
```

Comment:
Can we add a description here as well? Similar description as added in the opening comment of the PR.